### DISCUSSION TITLE
Setting the initial SOC for any experiment and plotting the SOC.

### DISCUSSION THREAD

**Discussion by kahnbiraj** [pybamm-team/PyBaMM#4198] [Q&A]
I am learning PyBaMM and trying to work with custom experiments. How can I start experiments at a preset SOC as a percentage of the nominal capacity ? I did find the simulation.build() function but could not understand how to start the experiment at a certain SOC in percentage. 
To plot the SOC later, I can use the Discharge capacity and the nominal capacity to compute the current SOC in percentage. But, I need to be able to set the initial SOC for that.

**Comment by TomTranter** [comment]
pass initial_soc to sim.solve(initial_soc=0.5) method. This uses the ElectrodeSoH class and is probably worth looking into the examples of and docs
